walk-in-interview/src/main/webapp/new-post.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+</head>
+
+<body>
+
+    <form>
+        <div class="header-container">
+            <button type="reset" onclick="location.href='/walk-in-interview/src/main/webapp/index.html'">Cancel</button>
+            <div>New Job Post</div>
+            <input type="submit" value="Post">
+        </div>
+        <div class="content-container">
+            <input type="text" class="job-field" id="new-job-title" name="job-title" placeholder="Job Title" required />
+            <textarea class="job-field" id="new-job-description" name="job-description"
+                placeholder="Describe the job here (responsibilities, preffered skills)..." required></textarea>
+            <input type="text" class="job-field" id="new-job-address" name="job-address" placeholder="Job Address" required/>
+            
+            <div class="job-field">
+                Requirements (for applicants):
+                <input type="checkbox" id="o-level" name="o-level" value="o-level">
+                <label for="o-level">O Levels</label>
+                <input type="checkbox" id="drivers-license" name="drivers-license" value="drivers-license">
+                <label for="drivers-license">Driver's License</label>
```

Comment:
Please make all the `name` for this checklist as `requirement-list`, then i can get the ticked value together in the servlet